pkg/client/client.go

```diff
@@ -31,7 +40,7 @@ type TakClient struct {
 	cancel   context.CancelFunc
 }
 
-func NewTakClient(ctx context.Context, config ClientConfig) (*TakClient, error) {
```

Comment:
Idk if it's a JetBrains Goland thing or if it's a gofmt thing but I was getting lint warnings for `client.ClientConfig` whichever linter prefers struct names that don't start with package names so it reads better.